packages/dashboard-frontend/src/pages/GetStarted/updateDevfileMetadata.ts

```diff
@@ -23,13 +28,19 @@ export function updateDevfileMetadata(devfile: api.che.workspace.devfile.Devfile
         location: meta.links?.v2,
       },
     } : {
-        custom: {},
-      });
+      custom: {},
+    });
     const metadata = devfile.metadata;
+    if (meta && metadata.name) {
+      metadata.name = `${metadata.name}-${getRandomString(4).toLowerCase()}`;
```

Comment:
not really related to your PR but still, ideally we should tell which policies we support
![Screenshot_20210714_145951](https://user-images.githubusercontent.com/5887312/125618520-b77b0383-120f-4ffe-928c-799e427884e0.png)

Reply:
after I fixed the URL to has `peruser` (instead of perUser) and pressed enter(without refreshing a page), it created workspace but error is kept
![Screenshot_20210714_150049](https://user-images.githubusercontent.com/5887312/125618624-318f388f-8e32-4b76-8f90-7f64496e5d18.png)

Reply:
I have fixed it